File: WinPaletter/GUI/Elements/ContainersAndBoxes/TablessControl.vb
Imports System.ComponentModel

Namespace UI.WP

    <Description("TabControl but without tabs for WinPaletter UI")> Public Class TablessControl : Inherits Windows.Forms.TabControl
        Public Sub New()
            SetStyle(ControlStyles.ResizeRedraw, True)
            Me.DoubleBuffered = True
        End Sub

        Protected Overrides Sub WndProc(ByRef m As Message)
            If m.Msg = &H1328 AndAlso Not DesignMode Then
                m.Result = CType(1, IntPtr)
            Else
                MyBase.WndProc(m)
            End If
        End Sub

    End Class

End Namespace

File: WinPaletter/GUI/Elements/ContainersAndBoxes/TablessControl.cs
using System;
using System.ComponentModel;
using System.Windows.Forms;

namespace WinPaletter.UI.WP
{

    [Description("TabControl but without tabs for WinPaletter UI")]
    public class TablessControl : System.Windows.Forms.TabControl
    {
        public TablessControl()
        {
            SetStyle(ControlStyles.ResizeRedraw, true);
            DoubleBuffered = true;
        }

        protected override void WndProc(ref Message m)
        {
            if (m.Msg == 0x1328 && !DesignMode)
            {
                m.Result = (IntPtr)1;
            }
            else
            {
                base.WndProc(ref m);
            }
        }

    }

}